engine/runtime/src/main/java/org/enso/interpreter/runtime/error/DataflowError.java

```diff
@@ -74,7 +75,7 @@ public static DataflowError withDefaultTrace(
 
   /** Slow version of {@link #withDefaultTrace(State, Object, Node, HasContextEnabledNode)}. */
   public static DataflowError withDefaultTrace(Object payload, Node location) {
-    return withDefaultTrace(null, payload, location, HasContextEnabledNode.getUncached());
+    return withDefaultTrace(payload, location, HasContextEnabledNode.getUncached());
```

Comment:
The removal of this `null` is causing the [change in test behavior](https://github.com/enso-org/enso/pull/12233/files#r1943397059):
```
DataflowError.withDefaultTrace() (DataflowError.java:65)
DataflowError.withDefaultTrace() (DataflowError.java:78)
DivNode.doLong() (DivNode.java:29)
```
Previously the `withDefaultTrace(Object payload, Node location)` was actually always attaching the full stack trace. **That wasn't correct** and we need to update the test.